_data/simple-icons.json

```
@@ -520,6 +520,11 @@
             "hex": "FF6A00",
             "source": "https://www.alibabagroup.com/en/ir/reports"
         },
+        {
+            "title": "Alienware",
+            "hex": "000000",
+            "source": "https://www.dell.com/es-es/gaming/alienware"
```

```suggestion
            "source": "https://www.dell.com/en-us/gaming/alienware"
```

If we can't go for a language-less website, probably safest to go with an english one.